website/docusaurus.config.js

```diff
@@ -78,9 +100,11 @@ module.exports = {
                     excludeExternals: false,
                 },
                 sortSidebar: groupSort,
-                pathToCurrentVersionTypedocJSON: `${__dirname}/api-typedoc-generated.json`,
                 routeBasePath: 'api',
-                python: true,
+                pythonOptions: {
+                    pythonModulePath: path.join(__dirname, '../src/crawlee'),
+                    moduleShortcutsPath: path.join(__dirname, 'module_shortcuts.json'),
+                },
```

Comment:
The new unified API docs generating pipeline - we don't need to generate the json ourselves, just supply the path to the project (and the `module_shortcuts.json` file, which we might want to remove / integrate later, further simplifying the interface).